### DISCUSSION TITLE
difference between blending and stacking in pycaret

### DISCUSSION THREAD

**Discussion by Mohammedsaied89** [pycaret/pycaret#1686] [Q&A]
Can any one please tell me the difference between blending and stacking in pycaret

**Comment by Yard1** [comment]
Blending uses VotingClassifier/Regressor to average the output of models, while stacking uses StackingClassifier/Regressor to fit a meta model on the predictions of models.

**Reply by Mohammedsaied89** [reply]
But at the both can we use kfold in training set
